src/CMakeLists.txt

```diff
@@ -2,8 +2,13 @@
 # @author Simon Heybrock
 # Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory, NScD Oak Ridge
 # National Laboratory, and European Spallation Source ERIC.
+set(INC_FILES
+convert.h dataset.h md_zip_view.h dimensions.h unit.h variable.h except.h)
```

Comment:
If IDEs need this, then there is a fair number of files missing here -- we have some header-only ones.